src/util/serialize.h

```diff
@@ -38,6 +38,7 @@
 #elif !defined(BYTE_ORDER) && defined(__BYTE_ORDER)
 	#define BYTE_ORDER __BYTE_ORDER
 #endif
+#undef HAVE_ENDIAN_H
```

Comment:
leftover from testing?
might need a portable solution, e.g.: if `endian.h` is not suitable on iOS, don't define `HAVE_ENDIAN_H`